offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -152,42 +159,81 @@ class OfferToPDFConverter implements OfferExporter {
         // Let's clear the existing item template content first
         htmlContent.getElementById("product-items-1").empty()
         //and remove the footer on the first page
-        htmlContent.getElementById("grid-table-footer").remove()
+        //htmlContent.getElementById("grid-table-footer").remove()
```

Comment:
why did you remove this?

Reply:
Because it lead to a null exception. Apparently it can't find the grid-table-footer even if it's there. I'll check it out again thanks for pointing it out